### ISSUE TITLE
Freeze a detector's pretrained weights in training with mask

### ISSUE THREAD

**Issue by igygi** [ShuLiu1993/PANet#42]
Hi, 

I originally did not have ground truth masks for my dataset so I have a PANet detector trained using the e2e_panet_R-50_FPN_1x_det.yaml config file only. Now that I have masks, I'd like to reuse the detector's weights and just train the masking branch using the e2e_panet_R_50_FPN_2x_mask.yaml config. 

Is that possible? How do I do that? Thanks.
